extensions/rocksdb-repos/FlowFileRepository.h

```diff
@@ -108,25 +99,17 @@ class FlowFileRepository : public ThreadedRepository, public SwapManager {
 
  private:
   void run() override;
+  void initialize_repository();
 
   void runCompaction();
   void setCompactionPeriod(const std::shared_ptr<Configure> &configure);
 
-  bool ExecuteWithRetry(const std::function<rocksdb::Status()>& operation);
-
-  void initialize_repository();
-
-  std::thread& getThread() override {
-    return thread_;
-  }
-
+  std::filesystem::path checkpoint_dir_;
   moodycamel::ConcurrentQueue<std::string> keys_to_delete;
   std::shared_ptr<core::ContentRepository> content_repo_;
-  std::unique_ptr<minifi::internal::RocksDatabase> db_;
+  std::unique_ptr<rocksdb::Checkpoint> checkpoint_;
```

Comment:
was this added by accident? (and the `checkpoint_dir_` member)

Reply:
Good catch, had some troubles while rebasing, I left it there by accident, updated in 5affb58bde8cb1fd4ec2424f60064a53a48bf12c